charts/sda-svc/templates/api-secrets.yaml

```diff
@@ -6,11 +6,79 @@ kind: Secret
+  config.yaml: |-
+    api:
+      rbacFile: {{ template "secretsPath" . }}/rbac.json
+      {{- if .Values.global.tls.enabled }}
+      serverCert: {{ template "tlsPath" . }}/tls.crt
+      serverKey: {{ template "tlsPath" . }}/tls.key
+      {{- end }}
+    broker:
+    {{- if .Values.global.tls.enabled }}
+      caCert: {{ template "tlsPath" . }}/ca.crt
+      {{- if .Values.global.broker.verifyPeer }}
+      clientCert: {{ template "tlsPath" . }}/tls.crt
+      clientKey: {{ template "tlsPath" . }}/tls.key
+      {{- end }}
+    {{- end }}
+      exchange: {{ default "sda" .Values.global.broker.exchange }}
+      host: {{ required "A valid MQ host is required" .Values.global.broker.host }}
+      port: {{ default (ternary 5671 5672 .Values.global.tls.enabled) .Values.global.broker.port }}
+      prefetchCount: {{ default 1 .Values.global.broker.prefetchCount }}
+      password: {{ required "MQ password is required" (include "mqPassAPI" .) }}
+    {{- if ne "" ( default "" .Values.global.broker.serverName ) }}
+      serverName: {{.Values.global.broker.serverName }}
+    {{- end }}
+      ssl: {{ .Values.global.tls.enabled }}
+      user: {{ required "MQ user is required" (include "mqUserAPI" .) }}
+    {{- if .Values.global.tls.enabled }}
+      verifyPeer: {{ .Values.global.broker.verifyPeer }}
+    {{- end }}
+      vhost: {{ include "brokerVhost" . }}
+    db:
+    {{- if .Values.global.tls.enabled }}
+      caCert: {{ template "tlsPath" . }}/ca.crt
+      {{- if eq "verify-full" .Values.global.db.sslMode}}
+      clientCert: {{ template "tlsPath" . }}/tls.crt
+      clientKey: {{ template "tlsPath" . }}/tls.key
+      {{- end }}
+    {{- end }}
+      host: {{ .Values.global.db.host }}
+      database: {{ .Values.global.db.name }}
+      password: {{ required "DB user is required" (include "dbPassAPI" .) }}
+      port: {{ .Values.global.db.port }}
+      sslmode: {{ ternary .Values.global.db.sslMode "disable" .Values.global.tls.enabled }}
+      user: {{ required "DB user is required" (include "dbUserAPI" .) }}
+    inbox:
+      type: {{ required "Inbox storage type is not set" .Values.global.inbox.storageType }}
+    {{- if eq "s3" .Values.global.inbox.storageType }}
+      accessKey: {{ required "Accesskey required for inbox" .Values.global.inbox.s3AccessKey }}
+      bucket: {{ required "S3 inbox bucket not set" .Values.global.inbox.s3Bucket }}
+      {{- if and .Values.global.inbox.s3CaFile .Values.global.tls.enabled }}
+      caCert: {{ template "tlsPath" . }}/ca.crt
+      {{ end }}
+      chunksize: {{ default 15 .Values.global.inbox.s3ChunkSize }}
+      {{- if .Values.global.inbox.s3Port }}
+      port: {{ .Values.global.inbox.s3Port }}
+      {{- end }}
+      readypath: {{ .Values.global.inbox.s3ReadyPath }}
+      secretKey: {{ required "Secretkey required for inbox" .Values.global.inbox.s3SecretKey }}
+      url: {{ required "S3 inbox URL missing" .Values.global.inbox.s3Url }}
+    {{- else }}
+      location: /inbox/
```

Comment:
Again, why do we hard code `inbox.location` here instead of create an entry in `values.yaml`.
In the README for sda-sftp-inbox it described as 

```
INBOX_LOCATION      | /ega/inbox/        | Path to POSIX Inbox backend 
```